File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target){
        vector<int> answer;
        for(int x =0; x < nums.size(); x++){
            for(int y = nums.size()-1; y > x; y--){
                if(nums[y] == target - nums[x]){
                    
                    answer.push_back(y);
                    answer.push_back(x);
                }
            }
        }
    return answer;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        look = {}
        for x in range(len(nums)):
            if target - nums[x] in look :
                return look[target - nums[x]], x
            look[nums[x]] = x

         
        
        
        